src/cargo/core/resolver/mod.rs

```diff
@@ -260,16 +260,8 @@ fn activate_deps_loop(
                 .conflicting(&resolver_ctx, &dep)
                 .is_some();
 
-        let mut remaining_candidates = RemainingCandidates::new(&candidates);
-
-        // `conflicting_activations` stores all the reasons we were unable to
```

Comment:
this comment has a lot more to do with how `conflicting_activations` influences the entire loop, so I think it should stay  And not move with the `ConflictMap::new();`.